src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/PlacementRequestsTest.kt

```diff
@@ -741,6 +775,49 @@ class PlacementRequestsTest : IntegrationTestBase() {
       }
     }
 
+    @Test
+    fun `It searches using multiple criteria including CRU management area id where user is manager`() {
+      `Given a User`(roles = listOf(UserRole.CAS1_WORKFLOW_MANAGER)) { user, jwt ->
+        `Given an Offender` { offender1Details, inmate1Details ->
+          `Given an Offender` { offender2Details, _ ->
+
+            val apArea1 = `Given an AP Area`()
+            val apArea2 = `Given an AP Area`()
```

Comment:
i think for tests related to filtering on the CRU Management Area we should build the CRU Management Area here (instead of ap area) for clarity. You can use `Given an CAS1 CRU Management Area`()` to do this. This applies to all updated tests

Reply:
Updated 2 x tests to do this now.